resource/sass/components/index.scss

```diff
@@ -0,0 +1,6 @@
+@import "./common.scss";
+@import "./variables.scss";
```

Comment:
variables should not be included from index.scss. This file was only included from theme's variables file